app-editors/nano/nano-6.3.ebuild

```diff
@@ -91,3 +91,11 @@ src_install() {
 
 	use split-usr && dosym ../../bin/nano /usr/bin/nano
 }
+
+pkg_postrm() {
+	local e=$(. /etc/profile &>/dev/null; echo "${EDITOR}")
```

Comment:
${EROOT}?

Reply:
Good point. Presumably also unset the var before sourcing.